daq/topology.py

```diff
@@ -531,9 +550,10 @@ def _generate_port_acl(self, port):
         if target_mac:
             assert self._append_acl_template(rules, 'baseline'), 'Missing ACL template baseline'
             self._append_device_default_allow(rules, target_mac)
-            self._write_port_acl(port, rules, file_path)
         else:
-            self._write_port_acl(port, self._make_default_acl_rules(), file_path)
+            rules = self._make_default_acl_rules()
```

Comment:
don't assign directly to rules -- append?

Reply:
I just didn't want to change the functionality of what it was doing. I assume we send the appended rules only for the 'if' case.

Reply:
In this case I'd assume it was an oversight in the original code, and just append the rules (keeps the overall pattern of the function more consistent).  If it's a problem (will fail some test somewhere), then we can re-visit.

Reply:
Changed.